cmd/krew/cmd/root.go

```diff
@@ -30,11 +33,24 @@ import (
 	"sigs.k8s.io/krew/internal/installation"
 	"sigs.k8s.io/krew/internal/installation/receipt"
 	"sigs.k8s.io/krew/internal/receiptsmigration"
+	"sigs.k8s.io/krew/internal/updatecheck"
+	"sigs.k8s.io/krew/internal/version"
 	"sigs.k8s.io/krew/pkg/constants"
 )
 
+const (
+	upgradeNotification = `A newer version of krew is available (%s -> %s). 
+Run "kubectl krew upgrade" to get the newest version!
+`
```

Comment:
maybe remove newline and use println?

Reply:
Println is not possible due to the format string. I changed to a normal string instead.